pkg/RepoManages.go

```diff
@@ -499,3 +571,29 @@ func (impl RepoManagerImpl) RefreshGitMaterial(req *git.RefreshGitMaterialReques
 	}
 	return res, err
 }
+
+
+func (impl RepoManagerImpl) GetPrEventDataById(id int) (*git.PrData, error) {
+	webhookPRDataEvent, err := impl.webhookEventRepository.GetPrEventDataById(id)
+
+	if err != nil {
+		impl.logger.Errorw("error in getting pr event for Id ", "Id", id, "err", err)
+		return nil, err
+	}
+
+	PrData := &git.PrData{
```

Comment:
we are create PrData object from webhookPRDataEvent above also, should extract out this transformation and reuse it

Reply:
done